packages/terra-icon/src/IconHazardLow.module.scss

```
@@ -0,0 +1,16 @@
+@import './orion-fusion-theme/IconHazardLow.module';
+@import './clinical-lowlight-theme/IconHazardLow.module';
+
+/* stylelint-disable selector-class-pattern */
+:local {
+  .IconHazardLow {
+    circle {
+      fill: var(--terra-icon-hazard-low-circle-fill,#0092e0);
```

Build is failing here:
```
packages/terra-icon/src/IconHazardLow.module.scss
 8:52  ✖  Expected single space after "," in a single-line function   function-comma-space-after
```
Easy fix, but was this generated? do we need to fix it somewhere else too?
```suggestion
      fill: var(--terra-icon-hazard-low-circle-fill, #0092e0);
```